bfcfg

```diff
@@ -1043,7 +1245,10 @@ boot_cfg()
   local l4proto l4proto_len
   local oob_mac_addr num oob_vlan_path
 
-  [ $dump_mode -eq 1 ] && return
+  if [ $dump_mode -eq 1 ]; then
```

Comment:
A few places use '$dump_mode -eq 1'.  You might want to check if we should use '$dump_mode -ne 0' to cover the '-all' case.